src/sql/Sql.hpp

```diff
@@ -1,10 +1,25 @@
 #pragma once
+#include <vector>
+#include <string>
 
-namespace bakcyl::sql {
+namespace bakcyl::sql 
+{
 
 class Sql {
 public:
     Sql();
+    //Products queries
+    std::vector<Product> selectAllProducts();
+    Product selectProduct(std::uint64_t productId);
```

Comment:
metody pobierające dane mogą być oznaczone jako metody const, ponieważ nie modydikują stanu obiektu.
Oczywiście w trakcie pracy może okazać się, że nie można tej metody oznaczyć jako const. Ale póki co możemy i powinniśmy to zrobić 

Product selectProduct(const std::uint64_t productId) **const**